misc/prerequisites.sh

```diff
@@ -13,7 +13,7 @@ OPUS_VERSION=1.3.1
 VPX_VERSION=1.11.0
 FDKAAC_VERSION=2.0.2
 NASM_VERSION=2.15.05
-FFMPEG_VERSION=5.0.1
+FFMPEG_VERSION=7.0
```

Comment:
Note: I have not tested the `prerequisites.sh` script, it would be great if one could do so.

Especially since AirenSoft provides a patch for enterprise users.

Reply:
I will not work, because some of the flags used in the `prerequisites.sh` script for building ffmpeg don't exist any more in current versions of ffmpeg.

Reply:
Alright, tested out `podman build -f Dockerfile.local`:
* `Unknown option "--disable-lzo"` Disabling this isn't allowed anymore https://github.com/FFmpeg/FFmpeg/commit/df272928ff8041f87b92f4cf8dbf12a28fe9cdf7
* `Unknown option "--enable-dct"` Couldn't find the commit
* `Unknown option "--enable-rdft"` Couldn't find the commit

Removed them in commit a027c9af.

Afterward ffmpeg seems to build. jemalloc then fails on my end:
```
+ curl -sSLf https://github.com/jemalloc/jemalloc/releases/download/5.3.0/jemalloc-5.3.0.tar.bz2
+ tar -jx --strip-components=1
tar: .appveyor.yml: Cannot change ownership to uid 66878, gid 100: Invalid argument
...
```

Adding `--no-same-owner` to the tar extraction works here (commit 6bf52f0d).

The build succeeds with those changes.